guide/14-deep-learning/add_custom_nlp_models_using_model_extension.ipynb

```diff
@@ -0,0 +1,658 @@
+{
```

Comment:
<p>This methods receives the parameters and their values, as requested in <code>getParameterInfo</code> . Additionally, the <code>kwargs</code>  has values that will determine the processing, such as batch size. You can alter these variables and return the value. </p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/2000/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='20'/>